tests/unit/get-series-articles.test.js

```diff
@@ -0,0 +1,27 @@
+import { getSeriesArticles } from '../../src/utils/get-series-articles';
+
+it('should correctly determine which series an article belongs to', () => {
```

Comment:
I wanted to write a test to best document logic, but didn't know where the best place to put it was. This directory was already configured for just so I put it here.